java/src/main/java/ai/rapids/cudf/ParquetWriterOptions.java

```diff
@@ -56,6 +61,14 @@ public Builder withTimestampInt96(boolean int96) {
       return this;
     }
 
+    /**
+     * Flattened precision values for all the decimal columns
```

Comment:
This needs to elaborate on what "Flattened' means.  Concrete examples would help.  Also it should explain what happens if the number of specified decimal precisions doesn't cover all decimal columns being written or left completely unspecified.